server/src/rest/api/models.py

```diff
@@ -66,22 +77,22 @@ class Legislation(models.Model):
     propublicaid = models.CharField(max_length = 9,default = "1")
 
     ##FEC api id
-    fecid = models.CharField(max_length = 9,default = "1")
+    fecid = models.CharField(max_length = 9,default = "ccccccccc")
```

Comment:
Also I don't want to screw with the migrations right before the demo. so lets merge and make a new pull request to fix the model.py